Fix docs in file-based access control (#20462)
trinodb/trino: docs/src/main/sphinx/security/file-system-access-control.md

These rules specify which users can access the system information management
interface. System information access includes the following aspects:

> - Read access to details such as Trino version, uptime of the node, and others

Reviewer: We should add that /v1/info and v1/status are always public and not affected by the rules

- Read access with the {doc}`/connector/system`.
- Write access to trigger {doc}`/admin/graceful-shutdown`.

> Some REST endpoints are always public and not affected by the rules, such as:

Reviewer: ```suggestion
The following REST endpoints are always public and not affected by these rules:
```
